js/parts-map/ColorAxis.js

```diff
@@ -558,6 +558,12 @@ extend(ColorAxis.prototype, {
         });
     },
 
+    // Define hasData function for non-cartesian seris.
```

Comment:
The comment is about series, but this is a colorAxis.
Also, there's another problem with comments - a repeated typo: `seris`